src/components/CacheFS.brs

```diff
@@ -23,7 +31,7 @@ function CacheFS() as Object
   prototype.read = function (key as String) as Object
     if (key = "") then return Invalid
 
-    filePath = m._PREFIX + m._hash(key)
+    filePath = m._PREFIX + m._directory + m._hash(key)
```

Comment:
Because user has to remember to put the slash character at the end and the code doesn't support nested directories (did you check if createDirectory can create nested folders?), I suggest switching to sth like "context" or "folder" instead of "directory" so it would accept the name of the folder, not the path